frontend-vite/.env.production

```diff
@@ -0,0 +1,4 @@
+VITE_API_SERVER_MATCHING_ENGINE=https://backend-matching-engine-lgw2vs6jea-uc.a.run.app/
```

Comment:
Don't check in environment variables like this as each deployment is different.
I would remove this file in the PR but make note of the structure in the README.